website/docs/components/dropdown/partials/code/component-api.md

```diff
@@ -3,16 +3,16 @@
 The Dropdown component is composed of different child components each with their own APIs:
 
 - The Dropdown component
-    - Optional header and footer
```

Comment:
Was this formatting change intentional? For some reason the way the markdown is parsed requires nested list items to have two two tabs instead of one (I know, a bit of a pain), but this will result in all of the items being in a single `<ul>`.

<img width="551" alt="Screenshot 2024-08-12 at 10 54 11 AM" src="https://github.com/user-attachments/assets/62878022-ca3e-4517-8e41-b08abdd5103f">

Reply:
My bad, I had a markdown linter running that did some accidental reformatting.